src/components/network/equipment-table.js

```diff
@@ -151,21 +157,34 @@ export const EquipmentTable = ({
     );
 
     const defaultCellRender = useCallback(
-        (cellData, numeric, fractionDigit, normed = undefined) => {
+        (
+            cellData,
+            numeric,
+            fractionDigit,
+            normed = undefined,
+            canBeInvalidated = undefined
+        ) => {
             const text = formatCell(cellData, numeric, fractionDigit, normed);
             return (
                 <TableCell
                     component="div"
                     variant="body"
                     style={{ width: cellData.width }}
-                    className={classes.tableCell}
+                    className={
```

Comment:
Maybe use clsx here to compose multiple style classes here

Reply:
Done (and also in single line diagram component, where it was originally not done)